sql/moz-fx-data-shared-prod/mozilla_org_derived/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Mozilla.org Derived
+description: |-
+  Mozilla.org data, usually derived from Google Analytics
+dataset_base_acl: derived
+user_facing: false
+labels: {}
+default_table_workgroup_access:
+- role: roles/bigquery.dataViewer
+  members:
+  - workgroup:mozilla-confidential
+workgroup_access:
```

Comment:
Table ACLs are additive to Dataset ACLs. Therefore, in order to restrict access to https://github.com/mozilla/bigquery-etl/pull/4557/files#diff-7d19dc15354cd1c33edcdb1e504d4617804c730316620813fcb43e800ccd3d4aR22 within this dataset the dataset ACL here has to be something more restrictive than `workgroup:mozilla-confidential` (like `[]`), and all other tables in the dataset that are `workgroup:mozilla-confidential` would need to be annotated specifically (or potentially automatically via `default_table_workgroup_access`).

Additionally the dataset base acl here being `derived` grants implicit read/write access to DE. That needs to be `derived_restricted` if it's not considered permissible for all of DE to have read/write access to any workgroup-confidential data in the dataset. In that configuration if anyone apart from Airflow needs direct write access to the dataset that would need to be granted here as well (https://mozilla-hub.atlassian.net/browse/DSRE-1023).

Reply:
Many thanks, indeed we'll lock down the stub attribution logs in this follow-up PR: https://github.com/mozilla/bigquery-etl/pull/4557